primitives/polkadot-core/Cargo.toml

```diff
@@ -7,6 +7,9 @@ edition = "2018"
 license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 
 [dependencies]
+parity-scale-codec = { version = "2.0.0", default-features = false, features = ["derive"] }
+parity-util-mem = { version = "0.9.0", default-features = false, features = ["primitive-types"] }
+hash256-std-hasher = { version = "0.15.2", default-features = false }
```

Comment:
Are you sure these are used anywhere?

Reply:
Only `parity_scale_codec` indeed. Probably some leftovers from some other approaches I've tried.